internal/key/svkeypair_test.go

```diff
@@ -102,6 +104,14 @@ func TestNewKMSKeypair(t *testing.T) {
 			},
 			expectErr: false,
 		},
+		{
+			name: "ED25519ph key",
```

Comment:
This test passes for me even without all the other changes, I'm not sure it's proving that a bug was fixed